forge/src/main/java/org/openjdk/skara/forge/PullRequestUtils.java

```diff
@@ -165,6 +165,7 @@ public static Hash targetHash(Repository localRepo) throws IOException {
     public static Repository materialize(HostedRepositoryPool hostedRepositoryPool, PullRequest pr, Path path) throws IOException {
         var localRepo = hostedRepositoryPool.checkout(pr.repository(), pr.headHash().hex(), path);
         localRepo.fetch(pr.repository().url(), "+" + pr.targetRef() + ":prutils_targetref", false);
+        localRepo.fetch(pr.repository().url(), "+" + pr.sourceRef() + ":prutils_sourceref", false);
```

Comment:
I'm curious as to why this is needed. Line 166 should make sure we are fetching the source/head of the pr. I don't see any use of the local branch `prutils_sourceref`.

Reply:
Sorry, I forget to delete it. I was thinking we need this and then I found it is useless, so I didn't use it any more. But I forget to delete it later.